oops/oops.go

```diff
@@ -72,6 +75,26 @@ func (e *oopsError) Error() string {
 	return b.String()
 }
 
+// CollectMetadata finds the first oopsError in err's chain and collects all metadata from oops errors in the chain.
```

Comment:
Please add that "If multiple oops errors in the chain set a metadata field, the outer-most one's value is used"